src/content/articles/en/how-to-slack.mdx

```diff
@@ -0,0 +1,187 @@
+
+### Use threads
+
+When someone posts something in a team channel and everybody starts responding to it you will definitely miss the question that someone else has asked in the same channel. To keep those conversations organized keep the chat about one topic in threads. Slack has the wonderful feature to share messages from the thread also into the channel if you think that the message might be interesting for others as well.
+
+### Answer with emojis
+
+Instead posting "I agree" or "Done" all the time, use an appropriate emoji for that. That will save you a lot of mess.
+
+### Respect time zones and DNDs
+
+In distributed teams, it is pretty likely that your team member will in another time zone that you are. Make sure that you keep that in mind when you send a message to people and trigger a notification. 10am at your place can be in the middle of the night for someone else.
+
+Same for DNDs (Do Not Disturb). If someone has snoozed notifications, just let them be off for that time.
+
+### Be visible
+
+Slack offers a small but very useful feature: Setting your status. You should use that feature to let people see if you are available or not. Being off for a lunch break, or being offline because you are going to the doc all these things can be shown with beautiful emojis. While you might also have a check-in/check-out channel (see below), having the status set makes it a lot easier for your colleague to know what you are at.
+
+### Have a face
+
+Working in a distributed teams also means that you won't see your colleagues that often. In teams spread over multiple time zones even less. That makes it important that you help your colleagues to remember you. If possible try to keep the video on when having calls. And even more important make sure that you have set up a profile picture. Nothing feels more awkward than chatting to someone not remembering who they are. Or meeting your colleague that you work with super closely at a team event and not recognizing them.
+
+## Organized communication
+
+In addition to introducing an etiquette, it is also important that you organize the Slack workspace in way that your employees can communicate in a structured way.
+
+Slack works in channels, so use that feature to organize your channels. Every new Slack workspace is created with the `#general` and `#random` channels but you will need way more than those two.
+
+### Team channels
+
+One of the first thing you might want to add are team channels. Depending on how your company is organized you will have channels for each department, such as `#team-engineering`, `#team-marketing` or `#team-sales`. If your teams are larger, you can even have more granular team channels: `#team-sales-uk` and `#team-sales-eu`
+
+It might even be interesting to have separate channels for teams of one. In a company no-one will work entirely on their own so people taking over different roles can join those channels as well.
```

Comment:
```suggestion
It might even be interesting to have separate channels for teams of one. In a company, no-one will work entirely on their own so people taking over different roles can join those channels as well.
```